lib/fluent/plugin/filter.rb

```diff
@@ -28,24 +28,57 @@ class Filter < Base
       include PluginLoggerMixin
       include PluginHelper::Mixin
 
+      class DuplicatedFilterMethodError < StandardError; end
```

Comment:
Will this error be rescued with this name by any code?
If not, there are no need to create a specific name for each errors, especially for BUG of plugins. See code to raise errors with "BUG: ..." messages in Fluentd.